spec/components/task_list_component/task_list_component_preview.rb

```diff
@@ -8,7 +8,16 @@ def default
           { task_name: "Edit the email address", path: "#", status: :cannot_start },
           { task_name: "Confirm the submission email address", path: "#", status: :cannot_start, active: false },
         ] },
-      { title: "do something else with a form",
+      { title: "Do something else with a form",
```

Comment:
"Do something else with a form" duplicates "Do yet another thing with a form" - was that deliberate?

Reply:
I wanted to see that the spacing between each section was consistent whether using body_text or rows, and just copied-pasted the existing example a bit more